src/Makefile

```diff
@@ -219,10 +219,15 @@ SIMU_OPTION += -DOPENMP
 # support yt inline analysis
 #SIMU_OPTION += -DSUPPORT_LIBYT
 
-# use patch group as the unit in libyt; this will speed up inline-analysis but increase memory consumption
+# use patch group as the unit in libyt, this will speed up inline-analysis but increase memory consumption
 # --> must enable SUPPORT_LIBYT
 #SIMU_OPTION += -DLIBYT_USE_PATCH_GROUP
 
+# support libyt interactive mode, this activate python prompt and does not shut down simulation when there are
+# errors in inline python script. must compile libyt with -DINTERACTIVE_MODE
+# --> must enable SUPPORT_LIBYT
+#SIMU_OPTION += -DSUPPORT_LIBYT_INTERACTIVE
```

Comment:
I would suggest simplifying the option name as `LIBYT_INTERACTIVE`, which also conforms with the naming convention `SUPPORT_SOFTWARE_NAME` adopted here.

Reply:
done